modules/compute/vm-instance/outputs.tf

```diff
@@ -28,3 +28,13 @@ output "internal_ip" {
   description = "Internal IP of the instances"
   value       = google_compute_instance.compute_vm[*].network_interface[0].network_ip
 }
+
+output "instructions" {
+  description = "Instructions on how to SSH into the created VM"
+  value       = <<-EOT
+  Use the following commands to SSH into the first VM created:
+    gcloud compute ssh --zone ${var.zone} ${google_compute_instance.compute_vm[0].name} --project ${var.project_id}
+  Or with tunnel through iap
```

Comment:
- use "IAP" or "Identity-Aware Proxy"
- I would present this as less of an "or" and give the reason: something like "if the machine is not accessible from the public internet, an SSH tunnel can be created through IAP" (you might consider something shorter)

Reply:
I have updated the wording to this effect.